docs/quickstart-eks.md

```diff
@@ -90,33 +91,21 @@ Else, If you are new to using BKPR on EKS, or if you want to create a new User P
 
 1. Browse to the [Cognito](https://console.aws.amazon.com/cognito/) module in the AWS Console
 2. Navigate to **Manage User Pools > Create a user pool**
-3. Enter a valid **Pool name**, like `eks-test`, then on the **Step through settings** button
-4. In the **Attributes** section leave all the checkboxes unticked as shown below and go to the **Next step**
+3. Enter a valid **Pool name**, like `eks-test`, then on the **Review defaults** button
```

Comment:
As mentioned on Slack, at least this needs to be kept in documentation:
> In the **Policies** section leave all the checkboxes unticked and select **Only allow administrators to create users** as shown below
Otherwise potentially anyone can create a new user and log in.

Reply:
Done. PTAL.